### PR TITLE
Use naming consistent with akka-http

### PR DESCRIPTION
**Pull request by notxcain**
#### As discussed in #17
1. Added proper package directories under `main/scala`
2. Deprecated `HttpMeterMetrics`in favor of `MeterDirectives`
3. Deprecated `HttpTimerMetrics`in favor of `TimerDirectives`
4. Deprecated recently added `StatusCodeMetrics` in favor of well-named `StatusCodeCounterDirectives`

### CODE REVIEW

File: src/main/scala/backline/http/metrics/StatusCodeMetrics.scala [hunk 1]
```diff
@@ -2,8 +2,7 @@ package backline.http.metrics
 
 import akka.http.scaladsl.server.Directive0
 
-trait StatusCodeMetrics extends MetricsBase
-    with HttpResponseWrapping {
+trait StatusCodeMetrics extends MetricsBase with StatusCodeCounterCommon {
```
Comment (adamdecaf) [inline src/main/scala/backline/http/metrics/StatusCodeMetrics.scala]: Should this trait be deprecated? Then it'll give a warning when it's imported by folks.

Comment (notxcain) [inline src/main/scala/backline/http/metrics/StatusCodeMetrics.scala]: Sure it should.

**Comment by adamdecaf** [comment]
This looks good aside from needing that deprecation. I can get 0.6.0 released today with that.

File: src/main/scala/backline/http/metrics/StatusCodeMetrics.scala (hunk 1 above)
Comment (notxcain) [inline src/main/scala/backline/http/metrics/StatusCodeMetrics.scala]: @adamdecaf done!

**Comment by notxcain** [comment]
@adamdecaf it would be great to have `0.6.0`, thanks.

**Comment by adamdecaf** [comment]
@notxcain 0.6.0 has been pushed, can you try it out? https://github.com/Backline/akka-http-metrics/tree/0.6.0

Also, I deleted the deprecated traits/methods in https://github.com/Backline/akka-http-metrics/commit/a7cfb2d1045a731cf93e98fb35e1a3ef8ece88b2 is that what you were thinking? 

The tree now looks pretty clean, thanks again! https://github.com/Backline/akka-http-metrics/tree/master/src/main/scala/backline/http/metrics

**Pull request merged**